public/static/docs/command-reference/import-url.md

```diff
@@ -48,15 +47,20 @@ determine whether the local copy is out of date.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type     | Discussion                                              | URL format                                 |
-| -------- | ------------------------------------------------------- | ------------------------------------------ |
-| `local`  | Local path                                              | `/path/to/local/file`                      |
-| `s3`     | Amazon S3                                               | `s3://mybucket/data.csv`                   |
-| `gs`     | Google Storage                                          | `gs://mybucket/data.csv`                   |
-| `ssh`    | SSH server                                              | `ssh://user@example.com:/path/to/data.csv` |
-| `hdfs`   | HDFS                                                    | `hdfs://user@example.com/path/to/data.csv` |
-| `http`   | HTTP to file with _strong ETag_ (see explanation below) | `https://example.com/path/to/data.csv`     |
-| `remote` | Remote path (see explanation below)                     | `remote://myremote/path/to/file`           |
+| Type     | Description                                           | `url` format                               |
+| -------- | ----------------------------------------------------- | ------------------------------------------ |
+| `local`  | Local path                                            | `/path/to/local/data`                      |
+| `s3`     | Amazon S3                                             | `s3://mybucket/data`                       |
+| `gs`     | Google Storage                                        | `gs://mybucket/data`                       |
+| `ssh`    | SSH server                                            | `ssh://user@example.com:/path/to/data`     |
+| `hdfs`   | HDFS to file\*                                        | `hdfs://user@example.com/path/to/data.csv` |
+| `http`   | HTTP to file with _strong ETag_ (explanation below)\* | `https://example.com/path/to/data.csv`     |
+| `remote` | Remote path (see explanation below)                   | `remote://myremote/path/to/data`           |
```

Comment:
Quick question @efiop. Only `import-url` supports remote:// URL notation? Asking because the `get-url` cmd ref doesn't have this line in the same table.

Reply:
Seems like will need another update per iterative/dvc/pull/3214

Reply:
I guess this is correct. I tried `dvc get-url remote://storage/get-started/data.xml` from inside my local clone of https://github.com/iterative/dataset-registry and got:
`ERROR: unexpected error - 'NoneType' object has no attribute 'repo'`

Reply:
this one looks like a bug we should create a ticket for in case you haven't done it yet

Reply:
I think it makes sense that `get-url` doesn't support `remote://` URLs since it's meant to be used from anywhere, not just from inside DVC repos. Maybe just the error message could be better? I did open an issue about that: iterative/dvc/issues/3236 I will ask a Q there to see if the behavior is correct in the first place.